magento-research/pwa-studio
# [Docs] Update dependencies

## Description

Bump dependency versions.
Switch out the `node-sass` dependency with `sass`.

## Related Issue

Closes #1235.

## Verification Steps

1. Navigate to `pwa-devdocs`
2. Run `npm audit`
3. Verify that there are no reported vulnerabilities.

## How Have YOU Tested this?

See **Verification Steps**

## Screenshots / Screen Captures (if appropriate):

n/a


## Checklist:
